src/app/home/home.component.ts

```diff
@@ -39,7 +39,7 @@ export class HomeComponent {
   onSubmit() {
     this.submitted = true;
     this.buttonText = 'Starting...';
-    this.homeService.saveInterested().subscribe(
+    this.homeService.saveInterested().first().subscribe(
```

Comment:
When chaining functions I've come to prefer putting them on separate lines for readability. It's not a deal breaker.

```suggestion
    this.homeService
      .saveInterested()
      .first()
      .subscribe(
```